test/fake_drbdsetup.sh

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env bash
```

Comment:
not sure if providing `fixture` is better then before. 
I was actually thinking to create data in file, but honestly imho it was better in the test itself so we have less data.

Btw why is this not json ?